lib/android.js

```diff
@@ -88,14 +88,25 @@ exports.detect = function detect(finished, sdkPath, ndkPath) {
 
 		function (next) {
 			exec('javac -version', function (err, stdout, stderr) {
-				if (!err) {
-					var m = stderr.match(/javac (.+)_(.+)/);
-					if (m) {
-						result.java.version = m[1];
-						result.java.build = m[2];
+				if (err) {
+					next();
```

Comment:
Should this be ```return next()``` to prevent the next line of code running or second invocation of next()?

Reply:
A code review from @muhammaddadu... What year is it? 😄